File: python/find-triangular-sum-of-an-array.py
class Solution:
    def triangularSum(self, nums: List[int]) -> int:
        tail = len(nums)
        while tail != 1:
            for i in range(1, tail):
                nums[i - 1] += nums[i]
            tail -= 1
        return nums[0] % 10

File: go/find-triangular-sum-of-an-array.go
func triangularSum(nums []int) int {
    for tail := len(nums); 1 < tail; tail-- {
        for i := 1; i < tail; i++ {
            nums[i - 1] += nums[i]
            nums[i - 1] %= 10
        }
    }
    return nums[0]
}
